src/layouts/core-dev-dot-layout/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { useRouter } from 'next/router'
 import { DatadogHeadTag, DatadogScriptTag } from 'lib/datadog'
 import Head from 'next/head'
+
```

Comment:
🙋🏻 Is this extra space intentional?

Reply:
It was, just to separate the imports here from the CSS import below.